src/theme/DocPage/index.tsx

```diff
@@ -60,68 +58,23 @@ function DocPageContent({
         tag: docVersionSearchTag(pluginId, version),
       }}
     >
-      <div className={clsx(styles.docPage, 'container')}>
-        {sidebar && (
-          <div
-            className={clsx(styles.docSidebarContainer, {
-              [styles.docSidebarContainerHidden]: hiddenSidebarContainer,
-            })}
-            onTransitionEnd={(e) => {
-              if (
-                !e.currentTarget.classList.contains(styles.docSidebarContainer)
-              ) {
-                return;
-              }
-
-              if (hiddenSidebarContainer) {
-                setHiddenSidebar(true);
-              }
-            }}
-            role="complementary"
-          >
-            <DocSidebar
-              key={
-                // Reset sidebar state on sidebar changes
-                // See https://github.com/facebook/docusaurus/issues/3414
-                sidebarName
-              }
-              sidebar={sidebar}
-              path={currentDocRoute.path}
-              sidebarCollapsible={
-                siteConfig.themeConfig?.sidebarCollapsible ?? true
-              }
-              onCollapse={toggleSidebar}
-              isHidden={hiddenSidebar}
-            />
-
-            {hiddenSidebar && (
-              <div
-                className={styles.collapsedDocSidebar}
-                title="Expand sidebar"
-                aria-label="Expand sidebar"
-                tabIndex={0}
-                role="button"
-                onKeyDown={toggleSidebar}
-                onClick={toggleSidebar}
-              >
-                <IconArrow />
-              </div>
-            )}
-          </div>
-        )}
-        <main className={styles.docMainContainer}>
-          <div
-            className={clsx(
-              'py-8 px-10',
-              styles.docItemWrapper,
-              {
-                [styles.docItemWrapperEnhanced]: hiddenSidebarContainer,
-              },
-            )}
-          >
-            <MDXProvider components={MDXComponents}>{children}</MDXProvider>
-          </div>
-        </main>
+      <div className={clsx('grid grid-cols-12 gap-x-5 mx-auto')} style={{ maxWidth: '1120px' }}>
```

Comment:
please leave a comment why maxWidth is needed

Reply:
I fix the width of the container like on mockup

Reply:
Docs page layout is broken on `480` screen:
1. Menu does not hide under burger menu anymore (compare to docusaurus)
2. TOC just disappears (again, compare to docusaurus page)
![image](https://user-images.githubusercontent.com/34244611/115684978-de833880-a368-11eb-9e77-94511abfc64a.png)

![image](https://user-images.githubusercontent.com/34244611/115685273-21dda700-a369-11eb-8f3b-a8900d716bc4.png)

Reply:
A also believe we should set `max-width` for containers via media queries, and not just use a static value.